source/examples/demo-stages/CMakeLists.txt

```diff
@@ -0,0 +1,261 @@
+
+#
+# External dependencies
+#
+
+find_package(OpenGL REQUIRED)
+find_package(GLM REQUIRED)
+find_package(glbinding REQUIRED)
+find_package(globjects REQUIRED)
+find_package(cppexpose REQUIRED)
+find_package(cppassist REQUIRED)
+
+
+#
+# Library name and options
+#
+
+# Target name
+set(target demo-stages)
+
+# Exit here if required dependencies are not met
+message(STATUS "Plugin ${target}")
+
+# Set API export file and macro
+string(MAKE_C_IDENTIFIER ${target} target_id)
+string(TOUPPER ${target_id} target_id)
+set(feature_file "include/${target}/${target}_features.h")
```

Comment:
I think this and the following two lines are not required for this example plugin.